iOS/Controllers/CalendarViewController.swift

```diff
@@ -82,15 +82,56 @@ class CalendarViewController: DayViewController {
 
         return self.calendarEvents.filter(inInterval: interval).map { $0.calendarKitEvent }
     }
+
+    override func dayViewDidSelectEventView(_ eventView: EventView) {
+        guard let eventDescriptor = eventView.descriptor as? Event else { return }
+        guard let userInfo = eventDescriptor.userInfo as? [String: String] else { return }
+        guard let id = userInfo["objectID"] else { return }
+        guard let event = self.calendarEvents.first(where: { $0.id == id }) else {
+            return
+        }
+
+        func clampMax(_ a: CGFloat, _ b: CGFloat) -> CGFloat { return min(a, b) }
```

Comment:
Why not use `min(a,b)`?

Reply:
I find `clampMax` to be more precise regarding my intention. I find myself making a lot of mistake with `min` and `max` when I really want a `clampMax` and `clampMin` respectively. In fact, I wrote that because I did that exact mistake again here.

Reply:
I guess I'm confused by the fact that `clamp` is most commonly used to restrict a value to a range (upper and lower bound) - see this [proposal](https://github.com/apple/swift-evolution/blob/master/proposals/0177-add-clamped-to-method.md). Whereas with your definition, the semantics become unclear IMHO. Which of the values is the upper bounds? Obviously both of them because that's the definition of `min` - take the smallest value.

I would advice to just go with using `min`.

Reply:
That's what im talking about, my use of clampMax is to give an upper bound to the value.